docs/source/deepinv.loss.rst

```diff
@@ -103,12 +103,18 @@ Metrics are generally used to evaluate the performance of a model. Some of them
 
 
 Transforms
```

Comment:
I think we can move this outside of Loss, ie to a new `deepinv.transform.rst` file